src/content/guides/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Ethereum guides
+description: A collection of practical guides explaining the basics of using Ethereum for beginners.
+lang: en
+---
+
+# Ethereum guides
+
+Do you want to start your Ethereum journey? It can be confusing to use and understand new technology. Our practical guides will teach you how to get started step-by-step.
+
+## Getting started
+
+1. [How to: "Register" an Ethereum account](/guides/how-to-register-an-ethereum-account/) - You will learn that you need a wallet app and the guide will show you where to find one.
+
+2. [How to: Use a wallet](/guides/how-to-use-a-wallet/) - An introduction to basic features of any wallet and how to use them.
+
+## Security basics
+1. [How to: Revoke smart contract access to your crypto funds](/guides/how-to-revoke-token-access/) - If you suddenly see a transaction in your wallet that you did not initiate, this guide will teach you how to prevent that from hapening again.
+
+## Using Ethereum
+1. [How to: Bridge tokens to layer 2](/guides/how-to-use-a-bridge/) - Are Ethereum transactions too costly? Consider moving to Ethereum scaling solutions called layer 2s. 
+2. [How to: Swap tokens](/guides/how-to-swap-tokens/) - Do you want to exchange your tokens for a different one? A simple guide how to do that.
```

Comment:
```suggestion
## Security basics

1. [How to: Revoke smart contract access to your crypto funds](/guides/how-to-revoke-token-access/) - If you suddenly see a transaction in your wallet that you did not initiate, this guide will teach you how to prevent that from hapening again.

## Using Ethereum

1. [How to: Bridge tokens to layer 2](/guides/how-to-use-a-bridge/) - Are Ethereum transactions too costly? Consider moving to Ethereum scaling solutions called layer 2s. 

2. [How to: Swap tokens](/guides/how-to-swap-tokens/) - Do you want to exchange your tokens for a different one? A simple guide how to do that.
```